### ISSUE TITLE
Threat can only be started once

### ISSUE THREAD

**Issue by silascutler** [silascutler/MalPipe#1]
Error while running.  

```
[TorNodes] Finished
Traceback (most recent call last):
  File "malpipe.py", line 23, in <module>
    malpipe.run()
  File "./MalPipe/malpipe/functions.py", line 41, in run
    schedule.run_pending()
  File "/usr/local/lib/python2.7/dist-packages/schedule/__init__.py", line 493, in run_pending
    default_scheduler.run_pending()
  File "/usr/local/lib/python2.7/dist-packages/schedule/__init__.py", line 78, in run_pending
    self._run_job(job)
  File "/usr/local/lib/python2.7/dist-packages/schedule/__init__.py", line 131, in _run_job
    ret = job.run()
  File "/usr/local/lib/python2.7/dist-packages/schedule/__init__.py", line 411, in run
    ret = self.job_func()
  File "/usr/lib/python2.7/threading.py", line 730, in start
    raise RuntimeError("threads can only be started once")
RuntimeError: threads can only be started once
```

**Comment by silascutler** [comment]
Running down possible solution.  Suspect `schedule` is starting the thread instance holding open the thread handle and not creating a new one.

**Comment by silascutler** [comment]
Problem almost fixed.  The fix will also include restarting of other crashed threads

**Comment by silascutler** [comment]
Fixed in #079ae17

**Comment by silascutler** [comment]
Fixed in 079ae17d7bb3c37982944455ce32287724a82db0
